<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionSettingsSidePanel</class>
 <widget class="QWidget" name="AcquisitionSettingsSidePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>871</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Roboto; font-size: 14px;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelTag">
     <property name="text">
      <string>Tag</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="acquisitionTag"/>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="appendDatetime">
     <property name="text">
      <string>Append datetime</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSampleSkip">
     <property name="text">
      <string>Sample skip</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sampleSkip">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelFrequency">
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="frequency">
     <property name="text">
      <string>10 MHz</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelBypassUL">
     <property name="text">
      <string>Bypass UL:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="bypassUL1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bypassUL2">
       <property name="text">
        <string>2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="limitRecords">
     <property name="text">
      <string>Limit records</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="recordCount">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="labelRecordLength">
     <property name="text">
      <string>Record length</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="recordLength">
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
     <property name="singleStep">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelTriggerMode">
     <property name="text">
      <string>Trigger mode</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="triggerMode">
     <item>
      <property name="text">
       <string>CONTINUOUS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SOFTWARE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LEVEL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>INTERNAL (Experimental)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EXTERNAL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QTabWidget" name="channelTabs">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="FileSaveSettingsPanel" name="fileSavePanel" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="ScopeUpdateSettingsPanel" name="scopeUpdatePanel" native="true"/>
   </item>
   <item row="9" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelPretrigger">
     <property name="text">
      <string>Horizontal shift</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="horizontalShift">
     <property name="minimum">
      <number>-32767</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
     <property name="singleStep">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QCheckBox" name="useMultipleChannels">
     <property name="text">
      <string>Use multiple channels</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSaveSettingsPanel</class>
   <extends>QWidget</extends>
   <header location="global">FileSaveSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScopeUpdateSettingsPanel</class>
   <extends>QWidget</extends>
   <header location="global">ScopeUpdateSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
